pkgs/development/python-modules/xpath-expressions/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, poetry
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "xpath-expressions";
+  version = "1.0.2";
+  disabled = pythonOlder "3.5";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0yn2n733cffk2pa7m1g3bp2kq8v9fa6dh91rxssly2sbpng0bk83";
+  };
+
+  nativeBuildInputs = [
+    poetry
```

Comment:
`poetry-core`. If it does not work with only `poetry-core`, then upstream needs to fix their dependency.